pkg/k8sops/generators.go

```diff
@@ -253,16 +167,16 @@ func (k *k8sops) GenerateMaps(kind string, svcCfg myspec.ServiceConfiguration) m
 }
 
 // GenerateService will produce resource configured according to the spec's
-// serviceConfiguration fields
-func (k *k8sops) GenerateService(svcCfg myspec.ServiceConfiguration) *v1.Service {
+// serviceConfiguration fields.
```

Comment:
interesting, periods ?

Reply:
I've been told that's our new convention: https://github.com/m3db/m3cluster/pull/233#discussion_r199624224